OneSchedule/ExceptionHandlingMiddleware/ExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Telegram.Bot.Types;
+using System.IO;
+using Telegram.Bot;
+using Microsoft.Extensions.Options;
+using OneSchedule.Settings;
+
+namespace OneSchedule
+{
+    public class ExceptionHandlingMiddleware
+    {
+        private readonly RequestDelegate _next;
+        private IOptions<TelegramSettings> _options;
+
+        public ExceptionHandlingMiddleware(RequestDelegate next, IOptions<TelegramSettings> options)
```

Comment:
Options were injected correctly, but you don't need to have a private field of that type, you need to get TelegramSettings value and assign it to private field